src/main/resources/com/mathworks/ci/freestyle/RunMatlabTestsBuilder/help-htmlReportArtifact.html

```diff
@@ -0,0 +1,12 @@
+<div>
+<br>
+Generate a test report in HTML format. By default, MATLAB names the artifact <b>testreportHTML</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace. 
+To override the default artifact name and location, specify a path relative to the project folder in the <b>File path</b> box. If the text box is empty, MATLAB does not generate an artifact.
+<br><br>
+Due to Jenkins Content Security Policy rules, the generated report might not open properly from within the Jenkins workspace. 
+Consider copying the report to a location outside the workspace and opening it from there. For more information, 
+see <a href="http://wiki.jenkins.io/display/JENKINS/Configuring+Content+Security+Policy">Configuring Jenkins Content Security Policy</a>.
+<br><br>
+<b>Note:</b> On macOS platforms, this feature is supported in MATLAB R2020b and later.
```

Comment:
I guess the corresponding help above has a different message:

This feature is not currently supported on macOS platforms.?